src/main/fc/fc_msp.c

```diff
@@ -1266,30 +1268,33 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
         }
         break;
 
-#if defined(VTX_COMMON)
+#if defined(USE_VTX_COMMON)
     case MSP_VTX_CONFIG:
         {
-            uint8_t deviceType = vtxCommonGetDeviceType();
-            if (deviceType != VTXDEV_UNKNOWN) {
+            vtxDevice_t *vtxDevice = vtxCommonDevice();
+            if (vtxDevice) {
 
-                uint8_t band=0, channel=0;
-                vtxCommonGetBandAndChannel(&band,&channel);
+                uint8_t deviceType = 0;
+                vtxCommonGetDeviceType(vtxDevice);
```

Comment:
This doesn't actually do anything.

Reply:
Nice catch. Thanks! I worked by chance because the configurator only checks for deviceType != 0xff and this was writing a zero. I’ll PR a fix tomorrow.